docs/release-notes/feature-node-selectors.rst

```diff
@@ -0,0 +1,14 @@
+:orphan:
+
+**New Features**
+
+-  Kubernetes Configuration: Allow cluster administrators to define Determined resource pools on
+   Kubernetes in terms of node selectors and/or affinities. Clusters administrators configure
+   resource pool's node selectors and/or affinities on the default pod spec level, under the
+   task_container_defaults.cpu_pod_spec or task_container_defaults.gpu_pod_specs. In this way,
+   cluster administrators can split a single cluster into several resource pools by way of node
+   labels.
+
+-  WebUI: For Determined clusters deployed on Kubernetes with multiple resource pools defined in
+   terms of node selectors and/or affinities, the resource pool slot counts correctly reflect the
+   state of the entire cluster. Slot counts and scheduling respect node selectors and affinities.
```

Comment:
This doesn't sound like a new feature? Is it a bug fix? if it's a new feature:

-  WebUI: Allow resource pool slot counts to reflect the state of the entire cluster. Allow slot counts and scheduling to respect node selectors and affinities. This impacts Determined clusters deployed on Kubernetes with multiple resource pools defined in
   terms of node selectors and/or affinities.

Reply:
it's a bug fix + expansion to a new feature -- let me try to rework this